go/internal/sidecar/sidecar.go

```
@@ -0,0 +1,536 @@
+				http.NotFound(w, r)
+				return
+			}
+			relayPublicWrite(w, r, cfg, writeRelay, path)
+		})
+	}
+	for _, path := range []string{
+		"/api/codex-auth/accounts/clear-cooldown",
+		"/api/codex-auth/reset-credits/consume",
+		"/api/oauth/accounts/clear-cooldown",
+	} {
+		path := path
```

golangci-lint (2.13.2)

[error] The copy of the 'for' variable "path" can be deleted (Go 1.22+)

(copyloopvar)